src/styles/blocks.scss/header-nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+    text-align: center;
+    list-style: none;
+    text-decoration: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    display: flex;
+    color: #060b35;
+    text-decoration: none;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    text-transform: uppercase;
+    line-height: 60px;
+    height: 60px;
+    margin-left: 20px;
+
+    &:hover {
+      color: $main-color;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+  margin-left: 0;
+  color: $main-color;
+
+  &::after {
+    content: '';
+    position: absolute;
+    display: block;
+    bottom: 0;
+    width: 100%;
+    height: 4px;
+    border-radius: 8px;
+    background: $main-color;
+  }
+}
```

Comment:
The `.is-active` class seems to be used to highlight the active link in the navigation. However, it's not a BEM modifier of the `&__link` class. It would be more appropriate to make it a modifier like `&__link--active` and put it inside the `&__link` block.